Beey.DataExchangeModel.Common/Project/ProjectErrorCategory.cs

```diff
@@ -17,21 +17,45 @@ public enum ProjectErrorCategory
 public enum ProjectErrorReason_Upload
 {
     Unknown,
+    MissingFile,
 }
 
 public enum ProjectErrorReason_Transcoding
 {
-    Unknown,
+    UnknownDuration,
+    MissingAudio,
+    MissingBitRateInformation,
+    NoAudioProduced,
+    NoVideoProduced,
+    TranscodingStuck,
+    UnsupportedMedia,
+    CorruptedMedia,
 }
 
 public enum ProjectErrorReason_Transcription
 {
-    Unknown,
+    NoDataToCreateTrsx,
+    InvalidTimestampInterval,
+    MissingMedia,
+    InvalidLanguage,
 }
 
-public enum ProjectErrorReason
+public enum ProjectErrorReason_General
 {
     Unknown,
     NotEnoughCredit,
     ServerShutdown,
 }
+
+// https://ffmpeg.org//doxygen/trunk/error_8h_source.html
+// codes are reversed, converted into their ascii counterparts and then to decimal
+// 
+// TODO move away
```

Comment:
Tohle by mělo být přímo v backendu, protože tyhle hodnoty zvenku nebudou nikdy potřeba.

Blbý, že nemáme třídu pro FFmpeg. Hele, tak do adresáře _ErrorHandling_ přidej statickou třídu s těmahle kódama. Tam by se to docela hodilo 🙂